presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -0,0 +1,191 @@
+=====================
+GeoSpatial Functions
+=====================
+
+Presto GeoSpatial functions support SQL/MM specification
```

Comment:
The first sentence is somewhat isolated from the rest of the doc.

btw `the SQL/MM specification.`